Foundation/include/Poco/LocalDateTime.h

```diff
@@ -85,13 +85,13 @@ class Foundation_API LocalDateTime
 		/// Creates a LocalDateTime from the UTC time given in dateTime,
 		/// using the time zone differential of the current time zone.
 
-	//@ deprecated
```

Comment:
@obiltschnig, @aleks-f: Some ctors/functions in this class seem not to have an alternative to be used. They are for example used in Dynamic/Var.h.

Is complete class deprecated?